roles/pulp_devel/templates/alias.bashrc.j2

```diff
@@ -239,7 +242,6 @@ pminio(){
         mc config host add s3 http://localhost:9000 AKIAIT2Z5TDYPX3ARJBA fqRvjWaPU5o0fCqQuUWbj9Fainj2pVZtBCiDiieS --api S3v4
         mc config host rm local
         mc mb s3/pulp3 --region eu-central-1
-        workon pulp
```

Comment:
So this is not needed?

Reply:
obviosly not for the commands in that same if branch. And it always bugged me that running that command activated the venv for me.